common/models/role-mapping.js

```diff
@@ -17,6 +17,15 @@ module.exports = function(RoleMapping) {
   RoleMapping.APP = RoleMapping.APPLICATION = 'APP';
   RoleMapping.ROLE = 'ROLE';
 
+  RoleMapping.getRelatedModels = function() {
+    if (!this.userModel) {
```

Comment:
This is weird... Why can't we just do this in `setup()`? I don't like that we aren't returning anything and just modifying the `RoleMapping` model.